src/Components/FavPage/FavPage.jsx

```diff
@@ -1,11 +1,34 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { FavCitiesList, Message } from '../';
+
 import styles from './FavPage.module.scss';
 
 function FavPage() {
-  return (
-    <main className={styles.container}>
-      <h2>Favourite Locations</h2>
-    </main>
-  );
+  const favCityList = useSelector((state) => state);
+
+  const listItems = favCityList.map((item) => {
+    return <FavCitiesList data={item} key={item.locationData.id} />;
+  });
+
+  function renderComponent() {
+    if (listItems.length !== 0) {
+      return (
+        <main className={styles.container}>
```

Comment:
you could just use react fragment instead of moving `main` tag :)
also no need to create separate `renderComponent` function

Reply:
Ok to use react fragment, but if I don't use renderComponent function how can I use if statement in the return statement of the FavPage Component? 

This does not work:
return (
  <main className={styles.container}>
    {
      if (listItems.length !== 0) { } else { } 
    }
  </main>);

Reply:
just use the same ternary operator you used before
listItems.length !== 0 ? (<>...</>) : (<>...</>)